src/components/Switch.js

```diff
@@ -1,24 +0,0 @@
-import React from "react"
```

Comment:
💯 , i assume this hasnt been used anywhere for a while now?

Reply:
It sure looks like it was part of an old set of UI requirements